app/helpers/application_helper.rb

```diff
@@ -41,7 +41,8 @@ def govuk_header_navigation_item(active_class: '', &block)
   end
 
   def search_active_class
-    'active' if controller_name == 'search' || (controller_name == 'sections' && action_name == 'index')
+    'active' if action_name == 'search' ||
```

Comment:
This feels a bit weird/horrible. I know it's what we're inheriting

Reply:
Chatted in slack about this. We've got quite a few dependency issues around layouts and controller action options that we can tidy up separately